lib/bom.js

```diff
@@ -812,8 +812,8 @@ declare class Headers {
     append(name: string, value: string): void;
     delete(name: string): void;
     entries(): Iterator<[string, string]>;
+    forEach((value: string, name: string, index: number) => any): void;
```

Comment:
The third callback argument is the Headers instance.

```js
new Headers({a:10, b:20}).forEach((a,b,c,d) => console.log(a,b,c,d))
// => 20 b Headers {  } undefined 
// => 10 a Headers {  } undefined
```

https://heycam.github.io/webidl/#es-forEach